properties.gradle

```diff
@@ -28,5 +28,5 @@ ext {
     appCliName = 'impexp'
 
     // documentation
-    docUrl = 'https://3dcitydb-docs.readthedocs.io/en/version-2022.1/'
```

Comment:
If we want to merge this to release 5.2 branch, we should use the release version of the doc `version-2022.1`

Reply:
`version-2022.2`, not `version-2022.1` right?

Reply:
👍